src/Dofs/ConstraintHandler.jl

```diff
@@ -90,11 +90,13 @@ struct ConstraintHandler{DH<:AbstractDofHandler,T}
     closed::ScalarWrapper{Bool}
 end
 
-function ConstraintHandler(dh::AbstractDofHandler)
+ConstraintHandler(dh::AbstractDofHandler) = ConstraintHandler(Float64, dh)
+
+function ConstraintHandler(T, dh::AbstractDofHandler)
```

Comment:
```suggestion
function ConstraintHandler(::Type{T}, dh::AbstractDofHandler) where T <: Number
```

Reply:
See https://docs.julialang.org/en/v1/manual/performance-tips/#Be-aware-of-when-Julia-avoids-specializing, even though it shouldn't matter here.